### PR TITLE
Make sure all written mail addresses are anchor links

### PR DESCRIPTION
**Pull request by ivarnakken**
# Description

I believe I went through all cases. One of them wasn't even correct

# Result

- [x] Changes look good on both light and dark theme.
- [x] Changes look good with different viewports (mobile, tablet, etc.).
- [x] Changes look good with slower Internet connections.

<table>
    <tr>
        <td>Before</td>
        <td>After</td>
    </tr>
    <tr>
        <td>
            
        </td>
        <td>
            
        </td>
    </tr>
</table>

# Testing

- [x] I have thoroughly tested my changes.

See image above

### CODE REVIEW

File: app/routes/brand/components/BrandPage.tsx [hunk 1]
```diff
@@ -11,7 +11,7 @@ const BrandPage = () => (
           Hei, vi i PR har laget noen retningslinjer for å hjelpe deg å bruke
           vårt brand og design. For å bruke logoen og designet på en måte som
           ikke er dekket av disse retningslinjene kontakt oss på{' '}
-          <a className={styles.inlineLink} href="mailto:abakus@abakus.no">
+          <a className={styles.inlineLink} href="mailto:pr@abakus.no">
```
Comment (norbye) [inline app/routes/brand/components/BrandPage.tsx:14]: 👀

**Review by norbye — APPROVED** [review]

**Review by falbru — APPROVED** [review]

File: app/routes/users/components/PhotoConsents.tsx [hunk 2]
```diff
@@ -52,9 +52,15 @@ const ConsentManager = ({
...
-            consent,
-          )} på ${presentableDomain}? Dersom du ønsker å fjerne noen spesifikke bilder, kan du i stedet sende en e-post til pr@abakus.no med informasjon om hvilke bilder du vil fjerne.`}
+          message={
+            <span>
+              Er du sikker på at du vil trekke bildesamtykket ditt for
+              {toReadableSemester(consent)} på {presentableDomain}? Dersom du
+              ønsker å fjerne noen spesifikke bilder, kan du i stedet sende en
+              e-post til <a href="mailto:pr@abakus.no">pr@abakus.no</a> med
+              informasjon om hvilke bilder du vil fjerne.
+            </span>
```
Comment (itsisak) [inline app/routes/users/components/PhotoConsents.tsx:62]: Seems like the message gets wrapped in a `span` tag anyways. I guess it does not hurt, but any reason for adding it here as well?

**Review by itsisak — APPROVED** [review]
Nice!

File: app/routes/users/components/PhotoConsents.tsx (hunk 2 above)
Comment (ivarnakken) [reply app/routes/users/components/PhotoConsents.tsx]: You're right, thats unnecessary! Removed it now - tested that it still work hehe

**Pull request merged**